src/KeyboardLayouts/ImprovedKeylayouts.h

```diff
@@ -53,7 +53,9 @@ enum KeyboardLeds : uint8_t {
     #define LAYOUT_US_ENGLISH
     #pragma message "Using default ASCII layout for keyboard modules"
```

Comment:
If we want to go this way, this pragma also needs a wrapper.

Reply:
I explained why I did it like this above, but I do not actually care about whether one or both branches get the ifndef, so I have updated the PR to add the 2nd wrapper as you said.

Reply:
(I have also changed it from HID_CUSTOM_LAYOUT_HIDE_PRAGMA_MESSAGE to HID_HIDE_LAYOUT_PRAGMA_MESSAGE so that the same define properly describes both #pragma messages - since the 1st pragma message is regarding the default layout, not a custom layout.)